Add Link Management chapter (#288)
TYPO3-Documentation/TYPO3CMS-Tutorial-Editors: Documentation/LinkManagement/ShortURLs/Index.rst

..  include:: /Includes.rst.txt

.. _short-urls:

==========
Short URLs
==========

> Short URLs are a simple and convenient way to share short or "tiny" URLs that point back to Pages in your TYPO3 installation.

Reviewer: Wrap lines at 80 chars where possible, see .editorconfig

Author: Good to know for next time.

> In addition, Short URLs can also point to Files such as documents and can also redirect to an external URL.

Reviewer: ```suggestion
Short URLs can point to files such as documents or redirect to external URLs.
```
Nitpick: Avoid unnecessary capitalization ("files"), and consider removing the repeated "also". F

Scenario - Shortening a long URL
================================

You want to share a link to your profile page found on your company website, however the URL itself is very long.

> With Short URLs you can generate a simple URL that can be shared easily.

Reviewer 2: It can also be kept when the target might change to something completely different

..  figure:: /Images/ManualScreenshots/LinkManagement/ShortURLCreate.png
    :alt: Screenshot of the Add Short URL button found within the Short URL submodule
    :class: with-border

General Tab
-----------

..  figure:: /Images/ManualScreenshots/LinkManagement/ShortURLGeneral.png
    :alt: Screenshot of the General Tab found inside Add Short URL.
    :class: with-border

:guilabel:`Short URL`: Select the domain you wish to link to Short URL. If you only have one site in your installation, there will be only one option to choose from. Then use
the dice icon to generate a random string of text. Note, you can also manually set the Short URL - however this is not recommended.

:guilabel:`Target`: Specify which page in the :guilabel:`Page Tree` your Short URL will point to. It is also possible to link to Files stored in TYPO3 and also point to external URLs.

:guilabel:`Force SSL Redirect`: Force all requests to be served via https.

Access Tab
----------

..  figure:: /Images/ManualScreenshots/LinkManagement/ShortURLAccess.png
    :alt: Screenshot of the Access Tab found inside Add Short URL.
    :class: with-border

:guilabel:`Enabled`: Toggle the Short URL on and off.

> :guilabel:`Publish Date`: Set a publish date.

Reviewer 2: see above
